infra/src/main/kotlin/fi/fta/geoviite/infra/publication/PublicationService.kt

```diff
@@ -543,14 +543,8 @@ class PublicationService @Autowired constructor(
             linkedTracksAndAlignments.map(Pair<LocationTrack, *>::first),
             validationVersions.locationTracks.map { it.officialId },
         )
-        val locationTrackErrors = validateSwitchLocationTrackReferences(
-            getLinkedTrackDraftsNotIncludedInPublication(
-                validationVersions,
-                linkedTracksAndAlignments.map { (t, _) -> t },
-            )
-        )
```

Comment:
Tämä oli käytännössä identtinen validoinnin kanssa, joka tehdään jo `referenceErrors`:in hakevassa polussa